src/symbolic/symbolic_value.ml

```diff
@@ -200,6 +200,7 @@ module I32 = struct
   let to_bool (e : vbool) =
     match view e with
     | Val (Num (I32 i)) -> Bool.const @@ Int32.ne i 0l
+    | Ptr _ -> Bool.const true
```

Comment:
Is this always correct? Can't the pointer be free'd at some point and become NULL? (I have no idea, I'm not familiar about this part, so I'm only asking to understand :-))

Reply:
You can free a pointer, but it will not become null. You have to explicitly set it to `(i32.const 0)` in Wasm, or `NULL` in C. 

Currently in concolic execution we will only have concrete pointer so a `Ptr _ <> 0` is always true

Reply:
Awww, sure, sorry!